homeassistant/components/tellduslive/strings.json

```diff
@@ -1,13 +1,13 @@
 {
   "config": {
     "abort": {
-      "already_configured": "TelldusLive is already configured",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
       "authorize_url_fail": "Unknown error generating an authorize url.",
       "authorize_url_timeout": "Timeout generating authorize url.",
```

Comment:
```suggestion
      "authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
```

Reply:
good catch !

Reply:
But does `authorize_url_timeout` need to be chaned to `oauth2_missing_configuration` becouse `authorize_url_timeout` is not in https://github.com/home-assistant/core/blob/b51a160cceea6d25680b90f9161fdf732a5888ab/homeassistant/strings.json#L65